lib/semian/sysv_state.rb

```diff
@@ -0,0 +1,59 @@
+require 'forwardable'
+
+module Semian
+  module SysV
+    class State < Semian::Simple::State #:nodoc:
```

Comment:
I made a couple of comments here, in general I think you can simplify this implementation by sharing more functionality from `Semian::Simple::State`. You may have to modify that class a little, but I think that's fine.